mediaapi/fileutils/fileutils.go

```diff
@@ -112,7 +112,11 @@ func WriteTempFile(reqReader io.Reader, maxFileSizeBytes config.FileSizeBytes, a
 	if err != nil {
 		return
 	}
-	defer (func() { err = tmpFile.Close() })()
+	defer (func() {
```

Comment:
same here. Maybe one of the best ways would be to just call `defer tmpFile.Close()`.